src/styles/article-card.css

```diff
@@ -0,0 +1,27 @@
+.card-grid{
+    display: grid;
+    grid-template-columns: auto auto auto;
+    /* justify-items: center; */
+}
+
+.post-card{
+    background-color: slategray;
+    margin:10%;
+    padding: 5%;
+    box-shadow: 2px 2px 2px 5px rgb(207, 203, 203);
+}
+.post-card:hover{
+    transform: scale(1.2) rotate3d(0, 1, 0, 360deg);
```

Comment:
This rotating of the article cards is quite confusing as a user, I like that the article card gets bigger but the rotating is a bit much.